maintenance/projects/js-toolkit/packages/liferay-npm-build-support/src/config.js

```diff
@@ -99,6 +103,22 @@ export function getSupportedLocales() {
 	return npmbuildrc.supportedLocales;
 }
 
+/**
+ * Get the `npm run start` webpack's directory
+ * @return {string}
+ */
+export function getStartDir() {
+	return npmbuildrc.start.dir;
```

Comment:
Do we need to add any optional chaining here or can we guarantee that these keys exist on the object?

Reply:
See https://github.com/liferay/liferay-frontend-projects/pull/526#discussion_r626361782